http-protocol-binding.md

```diff
@@ -213,16 +213,72 @@ and per [RFC7230, section 3][rfc7230-section-3], HTTP headers MUST only use
 printable characters from the US-ASCII character set, and are terminated by a
 CRLF sequence with OPTIONAL whitespace around the header value.
 
-String values MUST be percent-encoded as described in [RFC3986, section
-2.4][rfc3986-section-2-4] before applying the header encoding rules described in
-[RFC7230, section 3.2.6][rfc7230-section-3-2-6].
-
-When decoding an HTTP message into a CloudEvent, these rules MUST be applied in
-reverse -- [RFC7230, section 3.2.6][rfc7230-section-3-2-6] decoding to an ASCII
-string, and then a **single round** of percent-decoding as described in
-[RFC3986, section 2.4][rfc3986-section-2-4] to produce a valid UTF-8 String.
-(Note that applying percent-decoding an incorrect number of times can result in
-message corruption or security issues.)
+When encoding a CloudEvent as an HTTP message, string values MUST be
+percent-encoded as described below. This is compatible with
+[RFC3986, section 2.1][rfc3986-section-2-1] but is more specific
+about what needs encoding. The resulting string does not then need
+any further quoting, due to space and quote characters being encoded.
+
+When decoding an HTTP message into a CloudEvent, quoted strings as
+described in [RFC7230, section 3.2.6][rfc7230-section-3-2-6] MUST be
+decoded to an ASCII string, and then a **single round** of
+percent-decoding as described below. HTTP headers for CloudEvent
+attribute values do not support parenthetical comments, so the
+initial decoding only needs to handle double-quoted values, including
```

Comment:
it's not clear to me if the "including" here means we need to process backslash escapes or whether they're ignored. I'm guessing it means to NOT do them. 

Why are comment not allowed? Do we talk about that someplace else?

Reply:
First part: clarified that it means processing. (Basically that's part of "handling double-quoted values".)

See https://github.com/cloudevents/sdk-csharp/pull/117/files#diff-66bbdbd26048de2dc60bc1ae88c9626d4928326756b34dc187e90133b202af13 for some examples.

In terms of comments: I don't think we talk about that anywhere, but I don't think they ever *have* been allowed, and I'm asserting we don't want to introduce that capability. This is the problem with RFC 7230 section 3.2.6 - it expects the header field values to be defined elsewhere, and we basically haven't done that in the past.